### ISSUE TITLE
encoding error

### ISSUE THREAD

**Issue by xiaodaxia-2008** [phseiff/github-flavored-markdown-to-html#60]
when i use it to convert a markdown file, which contains some Chinese words, the following error was shown:

```
Traceback (most recent call last):
  File "MarkdownToHtml.py", line 14, in <module>
    f.write(gh_md_to_html.main(md_file))
  File "/home/rvbust/.local/lib/python3.7/site-packages/gh_md_to_html/__init__.py", line 834, in main
    html_content = core_converter(md_content)
  File "/home/rvbust/.local/lib/python3.7/site-packages/gh_md_to_html/__init__.py", line 476, in markdown_to_html_via_github_api
    requests.post("https://api.github.com/markdown/raw", headers=headers, data=markdown).content,
  File "/home/rvbust/.local/lib/python3.7/site-packages/requests/api.py", line 117, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/home/rvbust/.local/lib/python3.7/site-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/rvbust/.local/lib/python3.7/site-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/rvbust/.local/lib/python3.7/site-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/home/rvbust/.local/lib/python3.7/site-packages/requests/adapters.py", line 449, in send
    timeout=timeout
  File "/home/rvbust/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 706, in urlopen
    chunked=chunked,
  File "/home/rvbust/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 394, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/home/rvbust/.local/lib/python3.7/site-packages/urllib3/connection.py", line 234, in request
    super(HTTPConnection, self).request(method, url, body=body, headers=headers)
  File "/usr/lib/python3.7/http/client.py", line 1260, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/lib/python3.7/http/client.py", line 1305, in _send_request
    body = _encode(body, 'body')
  File "/usr/lib/python3.7/http/client.py", line 174, in _encode
    (name.title(), data[err.start:err.end], name)) from None
UnicodeEncodeError: 'latin-1' codec can't encode characters in position 3-4: Body ('安装') is not valid Latin-1. Use body.encode('utf-8') if you want to send it encoded in UTF-8.

```

the markdown file is very simple:
```
## 安装

查看 [安装指导](./static/InstallGuide.html)
```

**Comment by phseiff** [comment]
Hello, and thanks for rising this issue!

I will look into this this evening (CET).

In the meantime, have you tried using the `-o OFFLINE` option for conversion? It should be able to do the job for you in most cases, and might not raise this issue (requires you to use `pip3 install gh-md-to-html[offline_conversion]`, as stated in the README, though).

**Comment by phseiff** [comment]
This issue should be fixed in version `v1.21.1` now, to which you can update e.g. via `pip3 install gh-md-to-html>=1.21.1`.

**Comment by xiaodaxia-2008** [comment]
Thanks for your guide. I could generate the html successfully now.
